### ISSUE TITLE
Access to rawUpdates isn't synchronized

### ISSUE THREAD

**Issue by berserkk** [Arman92/go-tdlib#6]
`rawUpdates` can be accessed from multiple goroutines, but it's not synchronized.

https://github.com/Arman92/go-tdlib/blob/2f743ae341ab7b1fc6a43de1625ba76b46fdebf1/tdlib.go#L109-L112

https://github.com/Arman92/go-tdlib/blob/2f743ae341ab7b1fc6a43de1625ba76b46fdebf1/tdlib.go#L161-L164

Should use lock for it.
Here's related question on [Stack Overflow](https://stackoverflow.com/questions/21447463/is-assigning-a-pointer-atomic-in-golang).
Also, looks like it's possible to miss some updates, if `GetRawUpdatesChannel` was called with some delay.

**Comment by Arman92** [comment]
You're correct, also if GetRawUpdatesChannel get's called twice, the first channel will be lost.

I'll fix this as soon as I got the time.
